### DISCUSSION TITLE
Company Picker for CustomUserSettings

### DISCUSSION THREAD

**Discussion by ricocsharp** [OrchardCMS/OrchardCore#12164] [General]
Hello I need  to create a Company Picker(Select)  for using it, I also need to get this select's data from a Table in my Database created from previous Data Migration

Bests
Javier

**Comment by ns8482e** [comment]
CustomUserSetting is ContentType defined in orchard.

 You can create a custom part with custom display driver that read data from your tables and creates select in part’s shape.

you can add that part to new content type with customusersetting stereotype

**Reply by ricocsharp** [reply]
Thanks a lot for your answer its really helps, I have another question, I tried to create Data tables and "one to many" relations between them using the examples given in Dojo Course 2, then I realized that it is kind of deprecated so,  which is the way to stablish this relations and to create customa data tables if it is steel possible off course?

**Reply by hishamco** [reply]
I suggest to separate the second question into another thread to make things consistent

**Reply by ricocsharp** [reply]
Ok thnks I'll do it

**Comment by yassinsayed** [comment]
https://github.com/OrchardCMS/OrchardCore/discussions/9264
